src/object/wind.cpp

```diff
@@ -64,8 +68,9 @@ Wind::get_settings()
   result.add_float(_("Speed Y"), &speed.y, "speed-y");
   result.add_float(_("Acceleration"), &acceleration, "acceleration");
   result.add_bool(_("Blowing"), &blowing, "blowing", true);
+  result.add_bool(_("Affects Badguys"), &affects_badguys, "affectsbadguys", false);
```

Comment:
Can we name the lisp tag `affects-badguys` (we separate words with a dash in lisp by default).

Reply:
Updated as suggested.